b2/console_tool.py

```diff
@@ -1055,7 +1099,10 @@ def decode_sys_argv():
 
 def main():
     info = SqliteAccountInfo()
-    b2_api = B2Api(info, AuthInfoCache(info))
+    b2_http = B2Http()
```

Comment:
I think the API user should have `B2Http` constructor attach `ClockSkewHook()` by default. Then it could be disabled explicitly. As it is now, other API users will not get protected unless they look here and decide to attach the hook explicitly.